test/requirements-test.txt

```diff
@@ -2,7 +2,8 @@ accelerate
 azure-ai-ml
 azure-identity
 azure-storage-blob
-azureml-evaluate-mlflow>=0.0.60
+# azureml.evaluate.mlflow.hftransformers is deprecated in 0.0.66 and above
```

Comment:
Can you create a work item and add details about this? We may need to address in the future.

Reply:
Added work item 57818